src/main.cc

```diff
@@ -175,10 +175,26 @@ int main(int argc, char **argv) {
   }
 
   if (debug) {
-    std::cerr << "loading " << gltf_path << std::endl;
+    std::cerr << "loading " << scene_path << std::endl;
   }
 
-  auto [meshes, mesh_instances] = LoadMeshesFromGLTF(gltf_path);
+  const std::string OBJ_EXTENSION = ".obj";
+  const std::string GLTF_EXTENSION = ".gltf";
+
+  MeshList meshes;
+  MeshInstances mesh_instances;
+  if (scene_path.size() >= OBJ_EXTENSION.size() &&
+      scene_path.substr(scene_path.size() - OBJ_EXTENSION.size())
+              .compare(OBJ_EXTENSION) == 0) {
+    std::tie(meshes, mesh_instances) = LoadMeshesFromOBJ(scene_path);
+  } else if (scene_path.size() >= GLTF_EXTENSION.size() &&
+             scene_path.substr(scene_path.size() - GLTF_EXTENSION.size())
+                     .compare(GLTF_EXTENSION) == 0) {
+    std::tie(meshes, mesh_instances) = LoadMeshesFromOBJ(scene_path);
+  } else {
+    std::cerr << "Scene path should be either ending in .gltf or .obj\n";
+    return -1;
```

Comment:
nit: assert instead (for consistency w/ the rest of the code)
```suggestion
    assert(false);
```